packages/core/contracts/financial-templates/common/FundingRateApplier.sol

```diff
@@ -75,26 +76,23 @@ contract FundingRateApplier is Testable, Lockable {
      * @notice Constructs the FundingRateApplier contract. Called by child contracts.
      * @param _initialFundingRate Starting funding rate multiplier.
      * @param _fpFinderAddress Finder used to discover financial-product-related contracts.
+     * @param _priceIdentifier Unique identifier for DVM price feed ticker for child financial contract.
      * @param _timerAddress Contract that stores the current time in a testing environment.
-     * @param _identifier Unique identifier for DVM price feed ticker for child financial contract.
      * Must be set to 0x0 for production environments that use live time.
      */
     constructor(
         FixedPoint.Unsigned memory _initialFundingRate,
         address _fpFinderAddress,
-        address _timerAddress,
-        bytes32 _identifier
-    ) public Testable(_timerAddress) nonReentrant() {
-        // TODO: Should we constrain `_initialFundingRate > x && < y`?
+        bytes32 _priceIdentifier,
+        address _timerAddress
+    ) public nonReentrant() {
         cumulativeFundingRateMultiplier = _initialFundingRate;
         fpFinder = FinderInterface(_fpFinderAddress);
-        lastUpdateTime = getCurrentTime();
-        identifier = _identifier;
-    }
+        priceIdentifier = _priceIdentifier;
 
-    /****************************************
-     *         INTERNAL FUNCTIONS           *
-     ****************************************/
+        timer = Timer(_timerAddress);
```

Comment:
an instance of timer contract to remove the need to make `FundingRateApplier` a `Testable`.

Reply:
Are there any consequences as a result of this? For example, what if a contract is FundingRateApplier but not FeePayer?

Reply:
I don't think there are any implications for this other than what you just described.